python/ray/data/read_api.py

```diff
@@ -103,23 +103,33 @@ def from_items(
     parallelism: int = -1,
     output_arrow_format: bool = True,
 ) -> MaterializedDataset:
-    """Create a dataset from a list of local Python objects.
+    """Create a :class:`~ray.data.Dataset` from a list of local Python objects.
+
+    Use this method to create small datasets for testing and exploration.
 
     Examples:
-        >>> import ray
-        >>> ds = ray.data.from_items([1, 2, 3, 4, 5]) # doctest: +SKIP
-        >>> ds # doctest: +SKIP
-        MaterializedDataset(num_blocks=5, num_rows=5, schema={item: int64})
-        >>> ds.take_batch(2) # doctest: +SKIP
-        {"item": array([1, 2])}
+
+        .. testcode::
+
+            import ray
+
```

Comment:
nit, these blank lines are not necessary, right? Look a bit too verbose.

Reply:
The blankline before `import ray` isn't rendered. Included a blankline after imports because that's conventional, but I guess that isn't strictly necessary.